pkg/templates/templates_suite_test.go

```diff
@@ -185,6 +185,81 @@ func setUp() {
 		panic(err.Error())
 	}
 
+	// sample Nodes to test Infra node lookups
+	nodea1 := corev1.Node{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Node",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "node-infra1",
+			Labels: map[string]string{
+				"node-role.kubernetes.io/infra": "",
+			},
+		},
+	}
+
+	nodea2 := corev1.Node{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Node",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "node-infra2",
+			Labels: map[string]string{
+				"node-role.kubernetes.io/infra":  "",
+				"node-role.kubernetes.io/worker": "",
+			},
+		},
+	}
+
+	nodeb := corev1.Node{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Node",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "node-storage",
+			Labels: map[string]string{
+				"node-role.kubernetes.io/infra":   "",
+				"node-role.kubernetes.io/storage": "",
+			},
+		},
+	}
+
+	nodec := corev1.Node{
```

Comment:
I don't think this used in the tests. We can delete this one. Correct me if i am wrong